packages/uswds-core/src/styles/mixins/helpers/checkbox-and-radio-colors.scss

```diff
@@ -132,7 +180,11 @@
         }
       }
       &:checked:disabled,
-      &:checked[aria-disabled="true"] {
+      &:checked[aria-disabled="true"],
+      &:indeterminate:disabled,
+      &:indeterminate[aria-disabled="true"],
+      &[data-indeterminate]:disabled,
+      &[data-indeterminate][aria-disabled="true"] {
         @include format-input {
```

Comment:
## Indeterminate checkbox styles alternate location

As mentioned above, we could add the styles from lines 150-160 to this block but the redefinition for `checked:disabled, &:checked[aria-disabled="true"]` isn't necessary. 

Let me know if we'd prefer to move them here!

Reply:
Makes sense to leave as-is if `checked:disabled, &:checked[aria-disabled="true"]` don't apply.